### PR TITLE
Refactor TestGetListedBots To Use A Real Postgres DB For The Test

### PR DESCRIPTION
**Pull request by AbdulWahab3181**
## Describe your changes

image

## Issue ticket number and link

Closes https://github.com/stakwork/sphinx-tribes/issues/1741

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

File: handlers/bots_test.go [hunk 1]
```diff
@@ -446,44 +446,58 @@ func TestGetBot(t *testing.T) {
...
 		var returnedBots []db.Bot
 		err = json.Unmarshal(rr.Body.Bytes(), &returnedBots)
 		assert.NoError(t, err)
 		assert.Equal(t, http.StatusOK, rr.Code)
 
 		var filteredBots []db.Bot
-		for _, bot := range returnedBots {
-			if !bot.Deleted && !bot.Unlisted {
-				filteredBots = append(filteredBots, bot)
+		for _, returnBot := range returnedBots {
```
Comment (elraphty) [inline handlers/bots_test.go:494]: @AbdulWahab3181 I don't think we need this line anymore since the data is coming from the DB

**Review by elraphty — CHANGES_REQUESTED** [review]

File: handlers/bots_test.go (hunk 1 above)
Comment (AbdulWahab3181) [reply handlers/bots_test.go]: @elraphty You are right. Removed.

**Comment by aliraza556** [comment]
@elraphty  @AhsanFarooqDev,
Same issue here :

image

**Comment by aliraza556** [comment]
Hi @AbdulWahab3181 I have fixed please check my PR

**Comment by elraphty** [comment]
> @elraphty @AhsanFarooqDev, Same issue here :
> 
> image

It has been fixed, it was caused by a duplicate ID error.

**Pull request merged**